content/07.pathogenesis.md

```diff
@@ -76,6 +80,10 @@ After interventions, R~t~ decreased by an average of 64% across all countries, w
 The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden (note that this is in part because Sweden did not implement social distancing measures on the same scale as other countries).
 The study concludes that while large changes in R~t~ are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R~t~ below 1.
 
+More generally, population level epidemic dynamics can be both observed and modelled.
+Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to try to make predictions on possible future dynamics, such as for capacity needs or for the comparison of predicted possible outcomes of different prevention and control strategies.
+Beyond model predictions related to cases and deaths, infectious disease population models can also incorporate the dynamics of finer details such as possible genetic variations in pathogens or sometimes even possible evolutionary changes in pathogens due to relatively large numbers of pathogens which can evolve relatively quickly.
```

Comment:
```suggestion
Beyond model predictions related to cases and deaths, infectious disease population models can also incorporate the dynamics of finer details such as genetic variation among pathogens or even possible evolutionary changes in pathogens, due to relatively large numbers of pathogens which can evolve relatively quickly.
```
Whoo-hoo, my evolutionary biology background is finally useful! I am not sure I understand exactly what you mean by the last part of this sentence. My suspicion is that you mean that as even if mutation rate is extremely low, as you get a larger and larger population, that number becomes non-trivial and you are likely to see mutations arise. However, in terms of evolutionary change, these mutations would need to become more frequent at the population level, which depends on a number of factors that aren't really discussed here (I'm thinking specifically of mutation-selection balance and genetic drift). I'm happy to help clarify this text if you let me know more about what you mean!

Reply:
I do not really see what the point of this paragraph is? What is its central message? Not sure it belongs in this section?

Reply:
There were a few different points that may be would be better re-organized, I'll try to make it clearer. 1 point was to give an idea to non-epidemiologists & non-population-modelers what models do & how they interact with biological research. For the last part, I was hinting (but did not want to definitively say since it's a new infectious disease and requires quite a lot of computation) at a possible very interesting yet also looming thing to try to prevent which is that with very large numbers you can sometimes start to see pathogen evolution in populations in real time & this might for example occur in relation to therapeutics (drug resistance) or vaccines (e.g. ?immune escape? & also antigenic variation; there's starting to be some literature already on SARS-CoV-2 strains but I haven't yet gotten a chance to look at it closely so I don't know if it's related to antigens -- the idea that detailed spatial sequence data & population models may be used is very interesting, it happens for other infectious diseases though really I hope this pathogen won't get so complicated & maybe our control strategies can reduce such complications?? I really don't know but I also think it may require quite a deal of computation to answer so it may be too early to tell. That's related to what makes the science in real-time thing so unique!). Thanks for the feedback. This is working thing that'll get better with time but I thought these would be useful to have regardless.

Reply:
Since this is quite a lot (see comment above), I may have gotten ahead of myself because it's so interesting (yet also something to prevent). I will look up / re-read some literature mentions of this & see if I can find a more grounded description while also mentioning how this is just something that has been seen in other pathogens and we don't know how much it will be related to this pandemic.

Reply:
This seems really interesting but I wonder if there's another place in the paper where it would fit better. I think I remember seeing something about a potential evolution section... @rando2 was that discussed somewhere?

Reply:
copying this from my summary comment below: On looking again at this last sentence I added, I think I need to re-consider whether it is too new or different or early to include this for this pathogen/epidemic or, alternatively, if I happen to find/include some solid references where they mentioned it in the background or discussion at least or if I can make a non-flimsy connection to this type of research for other pathogens/epidemics; I know there's at least some stuff related to this but I see that I didn't write the sentence carefully enough to include as is / yet, and it may have been better to just maybe put it in as an Issue (rather than pull request) as a possible interesting topic.

Reply:
@rdvelazquez there have been several people who expressed interest in developing that section, but I haven't seen anyone come back with a PR yet! It's something that will definitely eventually be added since I am dying to look at the pangolin/cat/bat comparative genomics question!

Reply:
I think there are several points evo can be inserted @rdvelazquez @rando2 @yemarshall and this discussion is a bit of a mix. For instance,

1) Evolution analysis as in phylogenetic analysis and comparative genomics to show where SARS-CoV-2 falls in the virus tree. What makes SARS-CoV-2 more affective as a strain compared to SARS-CoV-1 - i.e. why is this a pandemic and can that be answered by genomic changes?

2) Evolution of SARS-CoV-2 since it was first discovered - the answer would be not much given a short time frame but how does it compare to what we know of other viruses? other RNA viruses? Can we try to beat the clock and move ahead of virus' evolution, if any?

3) Evolution of SARS-CoV-2 as it hopped between different species. Does it overlap what we see in 1? Can we predict 1 or 2 based on what we know of other zoonotic diseases from similar transmission potentials? Is there a comparative, attenuated strain infecting other species we can use for human vaccination?

and many more... I'd recommend sprinkling them across their respective, relevant bigger topics rather than having the pan-ultimate evo section, but that's just a personal preference.

Reply:
I think sprinkling the evolution discussions within their respective relevant topics could be best. If we find that we are repeating the same general evolution information (mutation rates, evidence of selection, phylogenetic reconstruction, etc.) as background in multiple places it might be worth pulling that all into one location at that point.

Reply:
I would be ok with removing this sentence from this pull request for now and then another pull-request could be opened adding in a more thorough discussion of how reproduction number and evolution overlap if that's desired.

Reply:
I also agree with @rdvelazquez that maybe simpler to just remove (at least for now) this last sentence I had suggested as an addition. (if later it makes sense to discuss population modeling which incorporates evolution then it can be added later; but after thinking it over better I can't actually say right now that it makes sense.) Would someone remove this sentence for me while I'm still learning this platform?

Reply:
```suggestion
```

Reply:
@yemarshall done